### DISCUSSION TITLE
"Maintain" buttons for warehouse resource transfer

### DISCUSSION THREAD

**Discussion by Ouaz** [Keriew/augustus#494] [General]
I don't know if someone is currently working on these buttons. If yes, do not take into account this suggestion.

Here's the _Maintain_ and _Stop_Maintain_ button png files (png transparent files, 30x19px):

- Maintain_1 and Maintain_2 : ![Maintain_1](https://user-images.githubusercontent.com/11946570/134977948-79b3788a-b345-4c35-bcfb-46e07e3d6cab.png) (normal & on hover)

- Maintain_3 : ![Maintain_3](https://user-images.githubusercontent.com/11946570/134978047-f2815bdd-495e-4d72-98a9-cc93ed5bd7dd.png) (on click)

- Stop_Maintain_1 and Stop_Maintain_2 : ![Stop_Maintain_1](https://user-images.githubusercontent.com/11946570/134978752-0ea3ddad-af02-4452-a923-158e718d7b37.png) (normal & on hover)

- Stop_Maintain_3 : ![Stop_Maintain_3](https://user-images.githubusercontent.com/11946570/134979004-887ca76d-90cd-473e-bbd6-97743a8ade09.png) (on click)

There's no _Maintain_4_ and _Stop_Maintain_4_ images... because I didn't understand how the code works, and therefore, how to create them. :laughing:

But in-game, it would look like this : ![maintain_warehouse](https://user-images.githubusercontent.com/11946570/134979646-ef566db0-e312-4f37-8caa-822d6b341c59.gif) (currently, by replacing the placeholder images, it's a mess, as all images stays displayed).

Maybe someone can make a proper button too, by adding the C3 built-in button borders and background (like the buttons used for land and sea trade policies in Trade Advisor Panel)

Sorry to have done only half the work, let me know what you think about it.

Thanks in advance for the answer.

**Comment by crudelios** [comment]
@Ouaz Do you have a discord account?

**Reply by Ouaz** [reply]
> Do you have a discord account?

It seems I should have one ^^

I never used Discord, so I created an account today: Ouaz#6549

**Comment by Ouaz** [comment]
I finished the buttons. In fact, I didn't notice that the placeholder images had a full-white background...  And me trying to create transparent png and some kind of layer masks... :rofl:

Whatever, it now looks good in-game:

![maintain_buttons](https://user-images.githubusercontent.com/11946570/135361220-ccc359f8-1f1f-42e8-b3dc-6bb61f800146.gif)

Here's the details:

- Maintain_1 ![Maintain_1](https://user-images.githubusercontent.com/11946570/135361281-477cd2d5-5a19-454c-b55b-eeab0fb78316.png) (normal)
- Maintain_2 ![Maintain_2](https://user-images.githubusercontent.com/11946570/135361329-f5cf3164-ee64-4871-903e-95c62f264a13.png) (on hover)
- Maintain_3 ![Maintain_3](https://user-images.githubusercontent.com/11946570/135361410-c83056f4-0931-49fc-8421-a0c60f877e6c.png) (on click)
- Maintain_4 ![Maintain_4](https://user-images.githubusercontent.com/11946570/135361441-7426fe61-0694-41a4-8663-ae2fb5e1e5c2.png) (disabled)
- Stop_Maintain_1 ![Stop_Maintain_1](https://user-images.githubusercontent.com/11946570/135361532-a560c0df-4d02-4e8d-a3ef-8cc12cfa37a1.png) (normal)
- Stop_Maintain_2 ![Stop_Maintain_2](https://user-images.githubusercontent.com/11946570/135361563-ae72cbbb-995e-4236-bdff-cdec8f0dd781.png) (on hover)
- Stop_Maintain_3 ![Stop_Maintain_3](https://user-images.githubusercontent.com/11946570/135361622-59ae5e21-d083-4438-95d4-ab5325c06f73.png) (on click)
- Stop_Maintain_4 ![Stop_Maintain_4](https://user-images.githubusercontent.com/11946570/135361723-57edf60b-5ad4-48b9-bb3b-1c1ef3372a8a.png) (disabled)

Here's the zipped file: [Maintain_Buttons.zip](https://github.com/Keriew/augustus/files/7255579/Maintain_Buttons.zip)

As always, let me know what you think about it (if you guys don't like it, not a big deal, I won't feel bad about that :yum: ), here or on Discord.
